src/components/Articles.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { fetchArticles } from '../api';
+
+export const Articles = () => {
+	const [articles, setArticles] = useState([]);
+
```

Comment:
Some loading state to display some feedback to the user that tharticles are loading would be a strong recommendation